src/mock-db-data/library-css/components/table-sortable.ts

```diff
@@ -98,7 +98,7 @@ export const TableSortable: LibraryExample[] = [{
   </table>
 </div>
 `,
-  codeMarkdown: `
+  codeHtml: `
```

Comment:
Trying to merge with `src/mock-db-data/library-css/components/toast.ts` and `tooltip.ts`breaks the build because they have `codeMarkdown`.